docs/Reply.md

```diff
@@ -391,3 +391,21 @@ For more details, see:
 - https://github.com/fastify/fastify/issues/1864 for the discussion about this feature
 - https://promisesaplus.com/ for the definition of thenables
 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then for the signature
+
+<a name="in addition"></a>
+### In addition
```

Comment:
I would call it `.raw` since there is not a brief paragraph that explains it in detail

Reply:
Make sense, i'll expand the section